src/utils/add_new.py

```diff
@@ -1,14 +1,20 @@
-from os import system
+from os import system, path
 
-import getFaces
+import sys, inspect
+current_dir = path.dirname(path.abspath(inspect.getfile(inspect.currentframe())))
+src_dir = path.dirname(current_dir)
+main_dir = path.dirname(src_dir)
+sys.path.insert(0, main_dir) 
```

Comment:
Why is this required?

Reply:
To import config from 2 directories upward. But I found a better way now. Will push it with the next commit shortly

Reply:
After installation, everything will be under the same directory. So, that will not be required.

Reply:
Oh yes you're right. That's why I've been getting failed tests. Thanks!